.github/workflows/macos.yaml

```diff
@@ -0,0 +1,203 @@
+name: macOS
+
+# To debug a job, add the following step:
+# - name: Debug with tmate
+#   uses: mxschmitt/action-tmate@v3
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths-ignore:
+      - '.github/workflows/linux.yaml'
+
+env:
+  COZY_DESKTOP_DIR: "${{ github.workspace }}/cozy-desktop"
+  COZY_STACK_STORAGE: "${{ github.workspace }}/storage"
+  COZY_URL: "http://localhost:8080"
+  COZY_PASSPHRASE: "cozy"
+  GOPROXY: "https://proxy.golang.org"
+  GO111MODULE: "on"
+  NO_BREAKPOINTS: ${{ 1 }}
+  NODE_ENV: "test"
+  NPM_CONFIG_PROGRESS: ${{ false }}
+  COZY_DESKTOP_HEARTBEAT: ${{ 1000 }}
+  DISPLAY: ":99.0"
+
+jobs:
+  unit:
+    runs-on: macos-latest
+    name: Unit tests
+    strategy:
+      matrix:
+        fs: ['APFS']
```

Comment:
Is is expected to have only `APFS` here?

Reply:
Yes it's expected. Unit and integration tests are not yet ready to be run on a HFS+ partition (mainly because some expectations need to differ).